### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jonthegeek**
Link to week4 slides and video.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +42,7 @@ Don't forget to check out the [Learning Objectives for each week](https://r4ds.g
 **3: How should I work on my package?** Chapters [5 ("Fundamental development workflows")](https://r4ds.github.io/bookclub-rpkgs/Companion/fundamental-development-workflows.html) & [18 ("Git and GitHub")](https://r4ds.github.io/bookclub-rpkgs/Companion/git-and-github.html)
 
 - Cohort 1 - [Tan Ho](https://https://twitter.com/_TanHo): [slides](https://r4ds.github.io/bookclub-rpkgs/Presentations/Week03/Cohort01/Development-Workflows-and-Git.html#1) | [video](https://youtu.be/DcV2km3Qx-g)
+
+- **4: How do I convert a script to a package?** Chapters [6 ("The package within")](https://r4ds.github.io/bookclub-rpkgs/Companion/the-package-within.html) & [11 ("Vignettes: long-form documentation")](https://r4ds.github.io/bookclub-rpkgs/Companion/vignettes.html)
+
+- Cohort 1 - [Jon Harmon](https://twitter.com/JonTheGeek): [slides](https://r4ds.github.io/bookclub-rpkgs/Presentations/Week04/Cohort01/week04.html#1) | [video](https://youtu.be/eMWgu9OQ0m8)
```
Comment (tanho63) [inline README.md:48]: slide link is dead. Looks like we have some inconsistencies in folder naming. Want to take a pass at those in this PR? Week 2 has Cohort01, Week 4 doesn't have a cohort folder at all

Comment (tanho63) [reply README.md:48]: (I should have caught the week 4 one before this lmao, myb)

Comment (jonthegeek) [reply README.md:48]: Oops, will fix!

Comment (jonthegeek) [reply README.md:48]: Ugh, can you merge this then I'll PR to fix? This doesn't have Week04 in it and I don't want to mess with it if I don't have to.

Comment (jonthegeek) [reply README.md:48]: Actually, lemme learn a thing, one moment...

**Pull request closed without merging**